package.json

```diff
@@ -7,9 +7,12 @@
     "warehouse-status": "bin/server"
   },
   "scripts": {
+    "pretest": "npm run swagger:doc",
     "test": "nyc --reporter=lcov --reporter=text mocha test/*.test.js",
     "lint": "eslint-godaddy *.js preboots/*.js routes/*.js test/*.js",
-    "pretest": "npm run lint"
+    "prepublishOnly": "npm run swagger:doc",
+    "swagger:doc": "swagger-jsdoc -d swagger/definition.js -o swagger/documentation.json",
```

Comment:
warehouse shoudl probally be internally consistent on what it calls this.

It's called just [`swagger` here](https://github.com/godaddy/warehouse.ai/pull/39/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R19)